src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -56,9 +53,8 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   const { user } = useUser();
 
   const isLoggedIn = !!user;
-  useConnectCurrentVenue();
 
-  const venue = useSelector(currentVenueSelectorData);
+  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
   const venueRequestStatus = useSelector(
     (state) => state.firestore.status.requested.currentVenue
   );
```

Comment:
@mike-lvov Note that by changing to `useConnectCurrentVenueNG` here, the not commonly used `venueRequestStatus` pattern below it will likely be invalid now.